scripts/pod_lib_lint.sh

```diff
@@ -38,7 +38,6 @@ fi
 # manually pushed to a temporary Specs repo. See
 # https://guides.cocoapods.org/making/private-cocoapods.
 #ALT_SOURCES="--sources=https://github.com/paulb777/Specs.git,https://github.com/CocoaPods/Specs.git"
```

Comment:
Out of scope, but we may want to use a Firebase fork of specs in the future instead of a personal repo.

Reply:
Agreed